src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -31,8 +31,11 @@ export function GaugeChartProvider(Private) {
       return function (selection) {
         selection.each(function (data) {
           const div = d3.select(this);
-          const width = verticalSplit ? $(this).width() : $(this).width() / data.series.length;
-          const height = (verticalSplit ? $(this).height() / data.series.length : $(this).height()) - 25;
+          const containerMargin = 5;
```

Comment:
not sure where this is defined. I'd add small comment here where this comes from.